src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -138,152 +139,24 @@ fun createListener(applicationState: ApplicationState, action: suspend Coroutine
                 log.error("Noe gikk galt", ex.cause)
             } finally {
                 applicationState.alive = false
+                applicationState.ready = false
```

Comment:
Vil det ha noen effekt å sette ready til false her..? Når du har satt alive til false vil jo kubernetes restarte podden uansett..?

Reply:
ja det vil ha noe å si for alle kafkalytterne, de vil da slutte å gjøre jobb